src/ImageSharp/Formats/Jpeg/GolangPort/OrigJpegConstants.cs

```diff
@@ -26,64 +26,32 @@ internal static class OrigJpegConstants
         public static readonly IEnumerable<string> FileExtensions = new[] { "jpg", "jpeg", "jfif" };
 
         /// <summary>
-        /// Represents high detail chroma horizontal subsampling.
+        /// Descibes the various header identifers for metadata profiles
         /// </summary>
-        public static readonly byte[] ChromaFourFourFourHorizontal = { 0x11, 0x11, 0x11 };
-
-        /// <summary>
-        /// Represents high detail chroma vertical subsampling.
-        /// </summary>
-        public static readonly byte[] ChromaFourFourFourVertical = { 0x11, 0x11, 0x11 };
-
-        /// <summary>
-        /// Represents medium detail chroma vertical subsampling.
-        /// </summary>
-        public static readonly byte[] ChromaFourTwoTwoVertical = { 0x11, 0x11, 0x11 };
-
-        /// <summary>
-        /// Represents low detail chroma vertical subsampling.
-        /// </summary>
-        public static readonly byte[] ChromaFourTwoZeroVertical = { 0x22, 0x11, 0x11 };
-
-        /// <summary>
-        /// Represents medium detail chroma horizontal subsampling.
-        /// </summary>
-        public static readonly byte[] ChromaFourTwoTwoHorizontal = { 0x22, 0x11, 0x11 };
-
-        /// <summary>
-        /// Represents low detail chroma horizontal subsampling.
-        /// </summary>
-        public static readonly byte[] ChromaFourTwoZeroHorizontal = { 0x22, 0x11, 0x11 };
-
-        /// <summary>
-        /// Describes component ids for start of frame components.
-        /// </summary>
-        internal static class Components
+        /// <remarks>
+        /// Encoding ASCII isn't available for NetStandard 1.1
+        /// </remarks>
+        internal static class ProfileIdentifiers
         {
             /// <summary>
-            /// The YCbCr luminance component id.
-            /// </summary>
-            public const byte Y = 1;
-
-            /// <summary>
-            /// The YCbCr chroma component id.
+            /// Describes the EXIF specific markers
             /// </summary>
-            public const byte Cb = 2;
+            public static readonly byte[] JFifMarker = { JFif.J, JFif.F, JFif.I, JFif.F, JFif.Null };
 
             /// <summary>
-            /// The YCbCr chroma component id.
+            /// Describes the EXIF specific markers
             /// </summary>
-            public const byte Cr = 3;
+            public static readonly byte[] IccMarker = { ICC.I, ICC.C, ICC.C, ICC.UnderScore, ICC.P, ICC.R, ICC.O, ICC.F, ICC.I, ICC.L, ICC.E, ICC.Null };
```

Comment:
I always forget, everything is NuGet package now. It should be very simple to get the ASCII bytes for these characters though:
```C#
static byte[] ToAsciiBytes(string str) => str.Select(c => (byte) c).ToArray();
```

I can add these changes this in the weekend.

Reply:
Yeah, thought just grabbing the package would be easy and I rushed a workaround so fixing it would be great! 

Only thing I'd like to do now is split the `ParseStream` method into two methods separating out the marker searching and the switch statement. 

I experimented with using the raw stream for all the bits outside the ScanDecoder but never got that working... We'd be able to associate the InputProcessor with the decoder only and possibly speed that portion of the code up. Another time perhaps.